internal/postprocessor/main.go

```diff
@@ -150,6 +149,13 @@ func (c *config) run(ctx context.Context) error {
 		log.Println("exiting post processing early")
 		return nil
 	}
+	// TODO(codyoss): In the future we may want to make it possible to be able
+	// to run this locally with a user defined remote branch.
+	modules, err := c.AmendPRDescription(ctx)
+	if err != nil {
+		return err
+	}
+	c.modules = append(c.modules, modules...)
```

Comment:
This feels odd here. The method that returns this has access to c already and could maybe be manipulated deeper?

Reply:
It would also be great if we could feed module to the modTidy command run. Those can be expensive calls too so limiting the processing to affected module would speed things up even more.